RxNimble.podspec

```diff
@@ -20,7 +20,7 @@ Pod::Spec.new do |s|
 
   s.subspec "Core" do |ss|
     ss.source_files  = "Source/Core/"
-    ss.dependency "Nimble", "~> 9.0"
+    ss.dependency "Nimble", "~> 9.2.0"
```

Comment:
This will require Nimble 9.2 and up, but won't accept 9.3 or higher. I think we want `~> 9.2` here, right?

Reply:
Great observation, I changed now ;) 
And update `Package.swift` too.